.github/actions/pre-install/action.yml

```diff
@@ -0,0 +1,11 @@
+runs:
+  using: composite
```

Comment:
Does this auto-run before each GH action workflow? Can you say more about it?

Reply:
Yup, Alex added it to the standard test workflow here https://github.com/inveniosoftware/workflows/blob/916417530bb4140fed8505da449592131e538e09/.github/workflows/tests-python.yml#L54. It's pretty slick, because it lets repos like this run additional processing without having to do a custom workflow file. It's also used for the Selenium install in https://github.com/inveniosoftware/pytest-invenio